sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -437,3 +437,288 @@ mod test_kubernetes_bootstrap_token {
+
+#[cfg(test)]
+mod test_reserved_resources_key {
+    use super::ReservedResourcesKey;
+    use std::convert::TryFrom;
+
+    #[test]
+    fn good_reserved_resources_key() {
+        for ok in &["cpu", "memory", "ephemeral-storage"] {
+            ReservedResourcesKey::try_from(*ok).unwrap();
+        }
+    }
+
+    #[test]
+    fn bad_reserved_resources_key() {
+        for err in &["", "cpa", ".bad", "bad.", &"a".repeat(64)] {
+            ReservedResourcesKey::try_from(*err).unwrap_err();
+        }
+    }
+}
+/// // =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+/// KubernetesQuantityValue represents a string that contains a valid kubernetes quantity value.
+/// https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct KubernetesQuantityValue {
+    inner: String,
+}
+
+impl TryFrom<&str> for KubernetesQuantityValue {
+    type Error = error::Error;
+
+    fn try_from(input: &str) -> Result<Self, Self::Error> {
+        ensure!(
+            KUBERNETES_QUANTITY.is_match(input),
+            error::Pattern {
+                thing: "Kubernetes quantity",
+                pattern: KUBERNETES_QUANTITY.clone(),
+                input
+            }
+        );
+
+        Ok(KubernetesQuantityValue {
+            inner: input.to_string(),
+        })
+    }
+}
+string_impls_for!(KubernetesQuantityValue, "KubernetesQuantityValue");
+
+#[cfg(test)]
+mod test_kubernetes_quantity_value {
+    use super::KubernetesQuantityValue;
+    use std::convert::TryFrom;
+
+    #[test]
+    fn good_kubernetes_quantity_value() {
+        for ok in &["129e6", "10Mi", "1024M", "1Gi"] {
+            KubernetesQuantityValue::try_from(*ok).unwrap();
+        }
+    }
+
+    #[test]
+    fn bad_kubernetes_quantity_value() {
+        for err in &["", "15%", ".bad", "bad.", &"a".repeat(64)] {
+            KubernetesQuantityValue::try_from(*err).unwrap_err();
+        }
+    }
+}
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+/// CpuManagerPolicy represents a string that contains a valid cpu management policy. Default: None
+/// https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct CpuManagerPolicy {
+    inner: String,
+}
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum ValidCpuManagerPolicy {
+    Static,
+    None,
+}
+
+impl TryFrom<&str> for CpuManagerPolicy {
+    type Error = error::Error;
+
+    fn try_from(input: &str) -> Result<Self, Self::Error> {
+        serde_plain::from_str::<ValidCpuManagerPolicy>(&input)
+            .context(error::InvalideCpuManagerPolicy { input })?;
+        Ok(CpuManagerPolicy {
+            inner: input.to_string(),
+        })
+    }
+}
+string_impls_for!(CpuManagerPolicy, "CpuManagerPolicy");
```

Comment:
There was some discussion of always enabling the `static` CPU manager policy. Where did that end up?

Reply:
I make `cpu-manager-policy` as configurable setting, which customers can configure it to `none` and `static`, and  I default `cpu-manager-policy` to `none` in `50-aws-k8s.toml`. 
```
cpu-manager-policy = "none"
```
I'll change it to `static`. Thanks

Reply:
Please check with @anguslees about https://github.com/bottlerocket-os/bottlerocket/issues/1140#issuecomment-781848321 - it's not clear to me that we need to expose a setting here, if "static" is good enough as a default.

Reply:
Yesterday I talked with Angus and we reached an agreement that `cpu-manager-policy` is really enabling a feature.  The default (none) is the legacy behaviour, and `static` turns on the feature. Therefore, I think we just need always enable the static CPU manager policy. I'll update `cpu-manager-policy`.